mlir-sycl/lib/Conversion/SYCLToLLVM/SYCLToLLVM.cpp

```diff
@@ -579,6 +588,9 @@ void mlir::sycl::populateSYCLToLLVMTypeConversion(
   typeConverter.addConversion([&](sycl::RangeType type) {
     return convertRangeType(type, typeConverter);
   });
+  typeConverter.addConversion([&](sycl::RoundedRangeKernelType type) {
```

Comment:
Noticing that code block at lines 582 - 684 should be moved after code blocks at lines 585-587.

Reply:
Not sure if this will get in, so created https://github.com/intel/llvm/pull/7710.